src/main/java/jenkins/branch/BranchSource.java

```diff
@@ -78,7 +77,12 @@ public SCMSource getSource() {
      * @return the strategy.
      */
     public BranchPropertyStrategy getStrategy() {
-        return strategy;
+        return strategy != null ? strategy : new DefaultBranchPropertyStrategy(new BranchProperty[0]);
```

Comment:
As `new DefaultBranchPropertyStrategy(new BranchProperty[0])` is effectively constant, couldn't we make it to `private static final` and use it as a default value?